lib/ansible/module_utils/rds.py

```diff
@@ -51,16 +51,18 @@ def __init__(self, dbinstance):
             self.name = None
         else:
             self.name = self.instance.get('DBInstanceIdentifier')
-        self.status = self.instance.get('DBInstanceStatus')
         self.data = self.get_data()
 
+    def status(self):
+        return self.instance.get('DBInstanceStatus')
```

Comment:
self.instance is really a dict, isn't it -  making this a function won't have any real effect, as far as I can tell - I could be wrong though.